db/crud.go

```diff
@@ -1441,7 +1441,7 @@ func (db *Database) getChannelsAndAccess(doc *document, body Body, revID string)
 			expiry = output.Expiry
 			err = output.Rejection
 			if err != nil {
-				base.InfofCtx(db.Ctx, base.KeyAll, "Sync fn rejected: new=%+v  old=%s --> %s", base.UD(body), base.UD(oldJson), err)
+				base.InfofCtx(db.Ctx, base.KeyAll, "Sync fn rejected doc %q: new=%+v  old=%s --> %s", base.UD(doc.ID), base.UD(body), base.UD(oldJson), err)
```

Comment:
Looking at this closer, this is rather verbose logging for Info (full doc bodies) - should switch the full bodies to debug, I think.  

I agree with the change to log docid, but I expect we should also add revid.

Reply:
Agreed, will fix that up - curious about the rev id - would we always have one at this stage?

Reply:
We're using the revid to look up the ancestor (to populate oldDoc), so generally yes.